src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   uploadImageHostname: "node.deso.org",
-  verificationEndpointHostname: "http://localhost:18001",
+  verificationEndpointHostname: "https://node.deso.org",
```

Comment:
Let's keep this as localhost. This ends up being the developer environment, so if we use node.deso.org calls to endpoints like `get-user-metadata` will fail.

Reply:
That file wasn't meant to be in this commit. I will revert it